template-compose/app/src/main/java/co/nimblehq/template/compose/ui/screens/home/HomeScreen.kt

```diff
@@ -1,24 +1,41 @@
 package co.nimblehq.template.compose.ui.screens.home
 
+import android.content.res.Configuration
 import androidx.compose.foundation.layout.padding
 import androidx.compose.foundation.layout.wrapContentHeight
-import androidx.compose.material.Text
-import androidx.compose.runtime.Composable
-import androidx.compose.ui.ExperimentalComposeUiApi
+import androidx.compose.material.*
+import androidx.compose.runtime.*
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.res.stringResource
 import androidx.compose.ui.text.style.TextAlign
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.hilt.navigation.compose.hiltViewModel
+import co.nimblehq.template.compose.ui.AppDestination
+import co.nimblehq.template.compose.R
 import co.nimblehq.template.compose.model.UiModel
 import co.nimblehq.template.compose.ui.theme.Dimension.SpacingNormal
-import co.nimblehq.template.compose.ui.theme.Theme
+import co.nimblehq.template.compose.ui.theme.ComposeTheme
 import timber.log.Timber
 
-@ExperimentalComposeUiApi
 @Composable
 fun HomeScreen(
+    viewModel: HomeViewModel = hiltViewModel(),
+    navigator: (destination: AppDestination) -> Unit
+) {
+    val uiModels: List<UiModel> by viewModel.uiModels.collectAsState()
+
+    HomeScreenContent(
+        title = stringResource(id = R.string.app_name),
+        uiModels = uiModels
+    )
+}
+
+@Composable
+private fun HomeScreenContent(
     title: String,
     uiModels: List<UiModel>
 ) {
-    Theme {
+    Surface {
```

Comment:
@luongvo May I know why we need to use `Surface` here 🤔  and FYI: When we run on the device, the result doesn't look like the `preview` and here's the result on my device:

![Screen Shot 2565-12-19 at 14 28 33](https://user-images.githubusercontent.com/28002315/208370986-36a591d4-b3d9-49c1-b249-a33e8705a4d5.png)

Not sure which one is your purpose, but if you prefer to make the content in the `center` of the screen, I think we can use the `Column` instead of `Surface.`

Reply:
@Wadeewee Nice spot 👍 Fixed in https://github.com/nimblehq/android-templates/pull/372/commits/caba80021fe63770d694cb0d196f7c9992e011df